### PR TITLE
vim25: Move internal stuff to internal package

### PR DESCRIPTION
**Pull request by anfernee**
Move internal methods/types to internal package.

### CODE REVIEW

**Comment by anfernee** [comment]
@dougm yeah, sure. that makes sense.

File: govc/host/esxcli/command.go [hunk 1]
```diff
@@ -21,7 +21,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/vmware/govmomi/vim25/types"
+	"github.com/vmware/govmomi/internal"
```
Comment (anfernee) [inline govc/host/esxcli/command.go]: @dougm if I change it to `github.com/vmware/govmomi/vim25/internal` then it's not importable from here in `github.com/vmware/govmomi/govc`

internal package spec described here: https://docs.google.com/document/d/1e8kOo3r51b2BWtTs_1uADIA5djfXhPT36s6eHVRIvaU/edit

Comment (anfernee) [reply govc/host/esxcli/command.go]: we can do `internal/vim25` though. it's not conflicting with methods/types, etc.

Comment (dougm) [reply govc/host/esxcli/command.go]: ok thanks @anfernee , let's stick with the top-level internal for now.

**Pull request merged**